pom.xml

```diff
@@ -40,6 +40,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-mongodb</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-crypto</artifactId>
+            <version>6.0.2</version>
```

Comment:
Can we change the versioning to interpolated variables?

Like so:
        <version.spring-security-crypto>6.0.2</version.spring-security-crypto>
        
        <version>${version.spring-security-crypto}</version>

Helps with maintainability